### PR TITLE
NMS-8295: Bridge topology is wrong on nodelink.jsp

### PR DESCRIPTION
**Pull request by rssntn67**
NMS-8295: bridge topology is wrong on nodelink.jsp

* JIRA: http://issues.opennms.org/browse/NMS-8295

I also update node.jsp for better layout of bridge elements

### CODE REVIEW

**Comment by rssntn67** [comment]
Alejandro, I introduced a NPE in enlinkdelementfactory....deleting I do not know why a check on lldpelement if it is not null....so now is green...but there are still other tests i want to run to be sure layout is right.

**Comment by rssntn67** [comment]
So I fixed the last issue on the page. Please merge this when you are ready!

Well there is a visualization problem when you have 30/40 ip addresses linked on the same mac....and the table is out of the page....but I'll work on this on the next bug fixing the topology map

File: opennms-webapp/src/main/webapp/element/node.jsp [hunk 1]
```diff
@@ -442,305 +442,130 @@ function confirmAssetEdit() {
...
+  <!--End Asset box, if info available --> 
 
   <!-- SNMP box, if info available -->
   <c:if test="${! empty model.node.sysObjectId}">
-    <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">SNMP Attributes</h3>
-    </div>
-    
+  <div class="panel panel-default">
```
Comment (j-white) [inline opennms-webapp/src/main/webapp/element/node.jsp]: Can we avoid reformatting the tables which we did not alter as part of these changes?

File: opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java [hunk 2]
```diff
@@ -317,61 +317,52 @@ public void delete(int nodeid) {
...
     public SharedSegment getHostSharedSegment(String mac) {
         
-        List<SharedSegment> segments = new ArrayList<SharedSegment>();
+        LOG.debug("getHostNodeSharedSegment: founding segment for mac:{}", mac);
+
+        final List<BridgeMacLink> links = new ArrayList<BridgeMacLink>();
+        m_bridgeMacLinkDao.findByMacAddress(mac).
+            stream().
+            filter(maclink -> maclink.getLinkType() ==  BridgeMacLinkType.BRIDGE_LINK)
+            .forEach(maclink -> {
```
[the remark this answers is not in the document]
Comment (rssntn67) [reply opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java]: Marcus....actually I was not able to reproduce it....I.m going to create an issue anyway

[the remark this answers is not in the document]
Comment (rssntn67) [reply opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java]: https://issues.opennms.org/browse/NMS-10258

File: opennms-webapp/src/main/webapp/element/node.jsp (hunk 1 above)
Comment (rssntn67) [reply opennms-webapp/src/main/webapp/element/node.jsp]: Jesse...this is very well verified and I use a more compact layout for the code
if you carefully look there only a compact representations of objects.....
but if you prefer I can redo the changes done

Comment (rssntn67) [reply opennms-webapp/src/main/webapp/element/node.jsp]: I'm going to make the requested changes....now

File: opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (rssntn67) [reply opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java]: I see that I answered a different question

[the remark this answers is not in the document]
Comment (rssntn67) [reply opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java]: Let me check this....

[the remark this answers is not in the document]
Comment (rssntn67) [reply opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/BridgeTopologyDaoHibernate.java]: This should work....        final List<BridgeMacLink> links =  m_bridgeMacLinkDao.findByMacAddress(mac).
                stream().filter(maclink -> maclink.getLinkType() ==  BridgeMacLinkType.BRIDGE_LINK).collect(Collectors.toCollection(ArrayList::new));

**Comment by j-white** [comment]
I reviewed the new layout with @rssntn67 last week. We should be good to merge once Bamboo goes green.

**Comment by rssntn67** [comment]
I have get your suggestions...please merge!

**Pull request merged**